### PR TITLE
Update Sample Applications to Work with the Migrated SDK

### LINKED ISSUE #37: Update Documentation for Sample Applications
**Is your suggestion related to a missing or misleading document? Please describe.**

Documentation for sample usage application should be updated describing the following,

- Installing the dependencies and [angular-cli](https://github.com/angular/angular-cli)
- Updating the `config.json` with configuration parameters
- Running the application with `ng serve`

**Describe the improvement**

Update the [README.md](https://github.com/asgardeo/asgardeo-auth-angular-sdk/tree/main/samples/basic-usage) for basic-usage sample application.

---

**Related Issues:** #16

### LINKED ISSUE #45: Add a Sample Application to Demonstrate Usage of the SDK without @angular/router
**Is your feature request related to a problem? Please describe.**

Developers should be able to quickly try out a sample application of the SDK which does not use angular router for in app navigation.

**Describe the solution you would prefer**

$subject

---

**Related Issues:** #17

### LINKED ISSUE #46: Refactor Style of Existing Samples to Match with Samples of SPA SDK
**Is your suggestion related to an experience ? Please describe.**

Sample applications should match the style with samples from [`asgardeo-auth-spa-sdk`](https://github.com/asgardeo/asgardeo-auth-spa-sdk/tree/main/samples) in order to provide a consistent experience.

**Describe the improvement**

$subject

---

**Related Issues:** #17

### LINKED ISSUE #53: Update Sample Applications to Work after @asgardeo/auth-spa Migration
**Is your suggestion related to an experience ? Please describe.**

Sample applications should be updated after the SDK is migrated from [`@asgardio/oidc-js`](https://www.npmjs.com/package/@asgardio/oidc-js)  to [`@asgardeo/auth-spa`](https://www.npmjs.com/package/@asgardeo/auth-spa).

**Describe the improvement**

$subject

---

Related Issues: #17, #38

### PR DESCRIPTION
**Pull request by JanithGan**
## Purpose

- Add `asgardeo-angular-app` sample (This sample does not use angular router)
- Change existing styles of the samples  to match with the samples from [`asgardeo-auth-spa-sdk`](https://github.com/asgardeo/asgardeo-auth-spa-sdk/tree/main/samples)
- Rename `basic-usage` sample with the name `asgardeo-angular-app-with-router`
- Add functions to check if the `Client ID` is provided
- Update sample functions to work with the migrated SDK
- Update documentation for sample applications

## Goals

Resolves #37, resolves #45, resolves #46, resolves #53

### CODE REVIEW

File: samples/asgardeo-angular-app-with-router/src/app/app.component.ts [hunk 1]
```diff
@@ -18,15 +18,22 @@
...
+    isConfigured: boolean;
 
-    constructor() { }
+    constructor() {
+        this.isConfigured = this.getClientIdStatus();
+    }
 
+    getClientIdStatus() {
+        if (authConfig.clientID === "") { this.isConfigured = false; }
+        else { return true; }
```
Comment (thivi) [inline samples/asgardeo-angular-app-with-router/src/app/app.component.ts:37]:
```suggestion
        if (authConfig.clientID === "") { 
            this.isConfigured = false; 
        } else { 
            return true; 
        }
```

File: samples/asgardeo-angular-app-with-router/src/app/home/home.component.html [hunk 2]
```diff
@@ -0,0 +1,65 @@
+<!--
+ Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (thivi) [inline samples/asgardeo-angular-app-with-router/src/app/home/home.component.html:2]:
```suggestion
 Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

File: samples/asgardeo-angular-app-with-router/src/app/home/home.component.html [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+    <ng-template #notSignedIn>
+        <div class="home-image">
+            <img
+                src="/assets/angular-logo.png"
+                class="logo"
+            />
+        </div>
+        <h3>
+            Sample demo to showcase how authenticate a simple client side application using <br />
```
Comment (thivi) [inline samples/asgardeo-angular-app-with-router/src/app/home/home.component.html:32]:
```suggestion
            Sample demo to showcase how to authenticate a simple client-side application using <br />
```

File: samples/asgardeo-angular-app-with-router/src/app/json-formatter/json-formatter.component.css [hunk 4]
```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (thivi) [inline samples/asgardeo-angular-app-with-router/src/app/json-formatter/json-formatter.component.css:2]:
```suggestion
 * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

File: samples/asgardeo-angular-app-with-router/src/app/json-formatter/json-formatter.component.html [hunk 5]
```diff
@@ -0,0 +1,18 @@
+<!--
+ Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (thivi) [inline samples/asgardeo-angular-app-with-router/src/app/json-formatter/json-formatter.component.html:2]:
```suggestion
 Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

File: samples/asgardeo-angular-app-with-router/src/app/json-formatter/json-formatter.component.ts [hunk 6]
```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (thivi) [inline samples/asgardeo-angular-app-with-router/src/app/json-formatter/json-formatter.component.ts:2]:
```suggestion
 * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

File: samples/asgardeo-angular-app-with-router/src/app/profile/profile.component.css [hunk 7]
```diff
@@ -0,0 +1,18 @@
+/**
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (thivi) [inline samples/asgardeo-angular-app-with-router/src/app/profile/profile.component.css:2]:
```suggestion
 * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

File: samples/asgardeo-angular-app-with-router/src/app/profile/profile.component.html [hunk 8]
```diff
@@ -0,0 +1,49 @@
+<!--
+ Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (thivi) [inline samples/asgardeo-angular-app-with-router/src/app/profile/profile.component.html:2]:
```suggestion
 Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

File: samples/asgardeo-angular-app-with-router/src/styles.css [hunk 9]
```diff
@@ -0,0 +1,194 @@
+/**
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (thivi) [inline samples/asgardeo-angular-app-with-router/src/styles.css:2]:
```suggestion
 * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

File: samples/asgardeo-angular-app/.eslintrc.js [hunk 10]
```diff
@@ -0,0 +1,95 @@
+/**
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (thivi) [inline samples/asgardeo-angular-app/.eslintrc.js:2]:
```suggestion
 * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

File: samples/asgardeo-angular-app/e2e/protractor.conf.js [hunk 11]
```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (thivi) [inline samples/asgardeo-angular-app/e2e/protractor.conf.js:2]:
```suggestion
 * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

File: samples/asgardeo-angular-app/e2e/src/app.e2e-spec.ts [hunk 12]
```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (thivi) [inline samples/asgardeo-angular-app/e2e/src/app.e2e-spec.ts:2]:
```suggestion
 * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

File: samples/asgardeo-angular-app/e2e/src/app.po.ts [hunk 13]
```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (thivi) [inline samples/asgardeo-angular-app/e2e/src/app.po.ts:2]:
```suggestion
 * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

File: samples/asgardeo-angular-app/e2e/tsconfig.json [hunk 14]
```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (thivi) [inline samples/asgardeo-angular-app/e2e/tsconfig.json:2]:
```suggestion
 * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

File: samples/asgardeo-angular-app/src/app/app.component.html [hunk 15]
```diff
@@ -0,0 +1,112 @@
+<!--
+ Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (thivi) [inline samples/asgardeo-angular-app/src/app/app.component.html:2]:
```suggestion
 Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

File: samples/asgardeo-angular-app/src/app/app.component.ts [hunk 16]
```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (thivi) [inline samples/asgardeo-angular-app/src/app/app.component.ts:2]:
```suggestion
 * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

File: samples/asgardeo-angular-app-with-router/src/app/json-formatter/json-formatter.component.html (hunk 5 above)
Comment (thivi) [inline samples/asgardeo-angular-app-with-router/src/app/json-formatter/json-formatter.component.html:2]:
```suggestion
 Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

File: samples/asgardeo-angular-app-with-router/src/app/json-formatter/json-formatter.component.ts (hunk 6 above)
Comment (thivi) [inline samples/asgardeo-angular-app-with-router/src/app/json-formatter/json-formatter.component.ts:2]:
```suggestion
 * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

File: samples/asgardeo-angular-app-with-router/src/app/profile/profile.component.css (hunk 7 above)
Comment (thivi) [inline samples/asgardeo-angular-app-with-router/src/app/profile/profile.component.css:2]:
```suggestion
 * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

File: samples/asgardeo-angular-app-with-router/src/app/profile/profile.component.html (hunk 8 above)
Comment (thivi) [inline samples/asgardeo-angular-app-with-router/src/app/profile/profile.component.html:2]:
```suggestion
 Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

File: samples/asgardeo-angular-app-with-router/src/styles.css (hunk 9 above)
Comment (thivi) [inline samples/asgardeo-angular-app-with-router/src/styles.css:2]:
```suggestion
 * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

File: samples/asgardeo-angular-app/.eslintrc.js (hunk 10 above)
Comment (thivi) [inline samples/asgardeo-angular-app/.eslintrc.js:2]:
```suggestion
 * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

File: samples/asgardeo-angular-app/e2e/protractor.conf.js (hunk 11 above)
Comment (thivi) [inline samples/asgardeo-angular-app/e2e/protractor.conf.js:2]:
```suggestion
 * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

File: samples/asgardeo-angular-app/e2e/src/app.e2e-spec.ts (hunk 12 above)
Comment (thivi) [inline samples/asgardeo-angular-app/e2e/src/app.e2e-spec.ts:2]:
```suggestion
 * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

File: samples/asgardeo-angular-app/e2e/src/app.po.ts (hunk 13 above)
Comment (thivi) [inline samples/asgardeo-angular-app/e2e/src/app.po.ts:2]:
```suggestion
 * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

File: samples/asgardeo-angular-app/e2e/tsconfig.json (hunk 14 above)
Comment (thivi) [inline samples/asgardeo-angular-app/e2e/tsconfig.json:2]:
```suggestion
 * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

File: samples/asgardeo-angular-app/src/app/app.component.html (hunk 15 above)
Comment (thivi) [inline samples/asgardeo-angular-app/src/app/app.component.html:2]:
```suggestion
 Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

File: samples/asgardeo-angular-app/src/app/app.component.ts (hunk 16 above)
Comment (thivi) [inline samples/asgardeo-angular-app/src/app/app.component.ts:2]:
```suggestion
 * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

File: samples/asgardeo-angular-app/src/app/app.module.ts [hunk 17]
```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (thivi) [inline samples/asgardeo-angular-app/src/app/app.module.ts:2]:
```suggestion
 * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

File: samples/asgardeo-angular-app/src/app/json-formatter/json-formatter.component.css [hunk 18]
```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (thivi) [inline samples/asgardeo-angular-app/src/app/json-formatter/json-formatter.component.css:2]:
```suggestion
 * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

File: samples/asgardeo-angular-app/src/app/json-formatter/json-formatter.component.html [hunk 19]
```diff
@@ -0,0 +1,18 @@
+<!--
+ Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (thivi) [inline samples/asgardeo-angular-app/src/app/json-formatter/json-formatter.component.html:2]:
```suggestion
 Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

File: samples/asgardeo-angular-app/src/app/json-formatter/json-formatter.component.ts [hunk 20]
```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (thivi) [inline samples/asgardeo-angular-app/src/app/json-formatter/json-formatter.component.ts:2]:
```suggestion
 * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

File: samples/asgardeo-angular-app/src/environments/environment.prod.ts [hunk 21]
```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (thivi) [inline samples/asgardeo-angular-app/src/environments/environment.prod.ts:2]:
```suggestion
 * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

File: samples/asgardeo-angular-app/src/environments/environment.ts [hunk 22]
```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (thivi) [inline samples/asgardeo-angular-app/src/environments/environment.ts:2]:
```suggestion
 * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

File: samples/asgardeo-angular-app/src/main.ts [hunk 23]
```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (thivi) [inline samples/asgardeo-angular-app/src/main.ts:2]:
```suggestion
 * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

File: samples/asgardeo-angular-app/src/polyfills.ts [hunk 24]
```diff
@@ -0,0 +1,82 @@
+/**
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (thivi) [inline samples/asgardeo-angular-app/src/polyfills.ts:2]:
```suggestion
 * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

File: samples/asgardeo-angular-app/src/styles.css [hunk 25]
```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (thivi) [inline samples/asgardeo-angular-app/src/styles.css:2]:
```suggestion
 * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

File: samples/asgardeo-angular-app/src/test.ts [hunk 26]
```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (thivi) [inline samples/asgardeo-angular-app/src/test.ts:2]:
```suggestion
 * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

File: samples/asgardeo-angular-app/tsconfig.app.json [hunk 27]
```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (thivi) [inline samples/asgardeo-angular-app/tsconfig.app.json:2]:
```suggestion
 * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

File: samples/asgardeo-angular-app/tsconfig.json [hunk 28]
```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (thivi) [inline samples/asgardeo-angular-app/tsconfig.json:2]:
```suggestion
 * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

File: samples/asgardeo-angular-app/tsconfig.spec.json [hunk 29]
```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (thivi) [inline samples/asgardeo-angular-app/tsconfig.spec.json:2]:
```suggestion
 * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

File: samples/asgardeo-angular-app-with-router/src/app/home/home.component.html (hunk 2 above)
Comment (thivi) [inline samples/asgardeo-angular-app-with-router/src/app/home/home.component.html:2]:
```suggestion
 Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

File: samples/asgardeo-angular-app-with-router/src/app/app.component.html [hunk 30]
```diff
@@ -0,0 +1,53 @@
+<!--
+Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (thivi) [inline samples/asgardeo-angular-app-with-router/src/app/app.component.html:2]:
```suggestion
Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

File: samples/asgardeo-angular-app-with-router/src/app/home/home.component.html (hunk 2 above)
Comment (JanithGan) [reply samples/asgardeo-angular-app-with-router/src/app/home/home.component.html:2]: Will be solved with a separate PR

**Review by thivi — APPROVED** [review]

**Pull request merged**